tests/Microsoft.Bot.Builder.LanguageGeneration.Tests/TemplatesTest.cs

```diff
@@ -786,45 +787,86 @@ public void TestLGResource()
         {
```

Comment:
break test cases into serveral case and make the test description more friendly to read like 

TestRepeatAdd
TestRepeatDelete
TestDeleteAll
TestUpdateWithWrongParamater
TestUpdateWithWrongReference